### PR TITLE
feat: clean up logging for agents

### PR DESCRIPTION
**Pull request by olbychos**

### CODE REVIEW

File: examples/tracing/agentops_tracing.py [hunk 1]
```diff
@@ -105,19 +109,37 @@ def current_session_ids(self):
...
+
+if __name__ == "__main__":
+    agent = get_react_agent()
     workflow = Workflow(
         flow=Flow(nodes=[agent]),
     )
     agentops_tracing = AgentOpsCallbackHandler(api_key=get_env_var("AGENTOPS_API_KEY"))
     workflow.run(
-        {"input": "explain the concept of quantum mechanics"},
+        {"input": "who win euro 2024"},
```
Comment (vitalii-dynamiq) [inline examples/tracing/agentops_tracing.py:143]: "Who won Euro 2024?"

File: dynamiq/nodes/agents/base.py [hunk 2]
```diff
@@ -254,15 +257,13 @@ def _run_llm(self, prompt: str, config: RunnableConfig | None = None, **kwargs)
                 **kwargs,
             )
             self._run_depends = [NodeDependency(node=self.llm).to_dict()]
-            logger.debug(f"Agent {self.name} - {self.id}: RAW LLM result:\n{llm_result.output['content']}")
             if llm_result.status != RunnableStatus.SUCCESS:
                 raise ValueError("LLM execution failed")
```
Comment (maksDev123) [inline dynamiq/nodes/agents/base.py:261]: Provide information about error in raised exception. `raise ValueError("LLM execution failed: {llm_result.output['content']}})`

File: dynamiq/nodes/agents/orchestrators/adaptive.py [hunk 3]
```diff
@@ -124,10 +122,7 @@ def get_next_action(self, config: RunnableConfig = None, **kwargs) -> Action:
         )
         self._run_depends = [NodeDependency(node=self.manager).to_dict()]
 
-        logger.debug(f"AdaptiveOrchestrator {self.id}: LLM response: {manager_result.output.get('content')}")
-
         if manager_result.status != RunnableStatus.SUCCESS:
-            logger.error(f"AdaptiveOrchestrator {self.id}: Error getting next action from Manager")
             raise ActionParseError("Unable to retrieve the next action from Manager.")
```
Comment (maksDev123) [inline dynamiq/nodes/agents/orchestrators/adaptive.py:126]: Same here

File: dynamiq/nodes/agents/orchestrators/linear.py [hunk 4]
```diff
@@ -118,10 +118,12 @@ def get_tasks(self, input_task: str, config: RunnableConfig = None, **kwargs) ->
             raise ValueError("Agent LLM failed to generate tasks")
```
Comment (maksDev123) [inline dynamiq/nodes/agents/orchestrators/linear.py:118]: Provide information about error in newly raise Exception.

**Comment by acoola** [comment]
@olbychos why did you change `debug` severity to `info`? In most cases, it looks like a debug (especially where we log results). Debug severity is controlled by ENV.

File: dynamiq/nodes/tools/firecrawl.py [hunk 5]
```diff
@@ -149,6 +144,6 @@ def execute(
                 "warning": scrape_result.get("data", {}).get("warning"),
             }
 
-        logger.debug(f"Tool {self.name} - {self.id}: finished with result {str(result)[:200]}...")
+        logger.info(f"Tool {self.name} - {self.id}: finished with RESULT:\n{str(result)[:200]}...")
```
Comment (acoola) [inline dynamiq/nodes/tools/firecrawl.py:147]: Why `RESULT` in upper case?

**Review by acoola — APPROVED** [review]

**Pull request merged**